src/libutil/file-system.cc

```diff
@@ -21,11 +22,18 @@ namespace fs = std::filesystem;
 
 namespace nix {
 
+
+static inline bool isAbsolute(PathView path)
```

Comment:
```suggestion
static bool isAbsolute(PathView path)
```

The "inline" keyword is rarely useful. The compiler will inline if it's useful.

Reply:
Sure